Makefile

```diff
@@ -22,9 +22,11 @@ SPHINXBUILDDIR ?= $(BUILD_DIR)/sphinx/html
 ALLSPHINXOPTS ?= -d $(BUILD_DIR)/sphinx/doctrees $(SPHINXOPTS) docs
 
 doc:
+	@ pip install -q Sphinx sphinx-rtd-theme
```

Comment:
we should add sphinx in the "requirements_dev.txt" at least

Reply:
I was wondering about that.  I'll add a commit.